apps/evm/src/libs/translations/translations/en.json

```diff
@@ -697,8 +692,7 @@
     },
     "promotionalTitle": "Boost your earnings by up to <GreenText>{{percentage}}</GreenText>",
     "stakeButtonLabel": "Stake XVS",
-    "tokensLeft_one": "<WhiteText>{{count}}</WhiteText> prime token left",
-    "tokensLeft_other": "<WhiteText>{{count}}</WhiteText> prime tokens left",
+    "tokensLeft": "TRANSLATION NEEDED",
```

Comment:
I think this is unintended

Reply:
Good catch. It seems `i18next-parser` is ignoring keys using pluralization. I've added them as comments to make sure they don't get overriden when calling the `extract-translation-keys` script.